benches/tpcc/install.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
This test requires tarantool-devel package. It would be nice to check that prerequisites are installed, otherwise show error message with a list of required packages. The same for other `install.sh` files.